### PR TITLE
Get rid of CoreUITests/CoreE2ETests targets

### PR DESCRIPTION
**Pull request by cobbal**
Clean up xcode project files 

- make sure directory structure matches between xcode and disk
- remove some duplicate things
- make sure every file is part of a target

refs: MBL-14548
affects: none
release note: none

[run-nightly]

### CODE REVIEW

**Comment by inst-danger** [comment]
## Latest Installable Builds

(will redirect to this page if no builds exist yet)
student: [Link](https://rebrand.ly/5xi6zoq) | [QR](https://qr.rebrandly.com/v1/qrcode?shortUrl=rebrand%2Ely%2F5xi6zoq)
teacher: [Link](https://rebrand.ly/i993bx9) | [QR](https://qr.rebrandly.com/v1/qrcode?shortUrl=rebrand%2Ely%2Fi993bx9)
parent: [Link](https://rebrand.ly/2cbny0x) | [QR](https://qr.rebrandly.com/v1/qrcode?shortUrl=rebrand%2Ely%2F2cbny0x)

**Comment by inst-danger** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  This pull request will not generate a release note.
  </td>
    </tr>
  </tbody>
</table>


Coverage | New % | Master % | Delta
-------- | ----- | -------- | -----
**Canvas iOS** | 93.31% | 93.33% | -0.02%

[MBL-14548](https://instructure.atlassian.net/browse/MBL-14548)

#### Affected Apps: None
<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

File: Student/StudentE2ETests/Modules/ModulesE2ETests.swift [hunk 1]
```diff
@@ -72,14 +71,14 @@ class ModulesE2ETests: CoreUITestCase {
         app.find(labelContaining: "This is a page for testing modules").waitToExist()
     }
 
-    func testLaunchIntoQuizModuleItem() {
+    func xtestLaunchIntoQuizModuleItem() {
```
Comment (vanwagonet) [inline Student/StudentE2ETests/Modules/ModulesE2ETests.swift:74]: Do we want this test disabled?

File: Student/StudentE2ETests/Quizzes/QuizzesE2ETests.swift [hunk 2]
```diff
@@ -18,33 +18,35 @@
 
 import XCTest
 @testable import TestsFoundation
-@testable import CoreUITests
 @testable import Core
 
 class QuizzesE2ETests: CoreUITestCase {
-    func testQuizQuestionsNoMoreNativeView() {
+    func testQuizQuestionsNoMoreNativeView() throws {
+        try XCTSkipIf(true, "Broke unnoticed at some point")
```
Comment (vanwagonet) [inline Student/StudentE2ETests/Quizzes/QuizzesE2ETests.swift:25]: Has this been broken on master then?

Comment (cobbal) [reply Student/StudentE2ETests/Quizzes/QuizzesE2ETests.swift:25]: Ticketed as MBL-14571, I'll address it in a future PR

**Pull request merged**